.run/dev-viv.run.xml

```diff
@@ -0,0 +1,25 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
Should we be committing this file?

Reply:
Yeah it is okay to include; it is just the Pycharm run configuration so it will load the Pycharm configurations properly if someone opens the project in Pycharm. It will set everything properly, and just need to update the python path to get it to run.
The `dev.run.xml` file works for Mac/Linux, this one is for Windows (Vivian specifically).

Reply:
Let's pick a better name in the future since both Dhruvin and Amolik will be using Windows too!

Reply:
While the Max/Linux files are the same for everyone, the Windows file seem to be different per person. Hence the name :)

Reply:
I will investigate if I can make a generic windows one though

Reply:
Ew windows why you like this. But okay, if that is possible that would be amazing!